### PR TITLE
Full CRACO

### PR DESCRIPTION
**Pull request by profxj**
as titled

### CODE REVIEW

File: papers/H0_I/Analysis/CRACO/Cloud/nautilus_craco_full_1.yaml [hunk 1]
```diff
@@ -41,9 +41,9 @@ spec:
             python setup.py develop;
             cd ../zdm;
             git fetch;
-            git checkout H0_paper;
+            git checkout full_craco;
```
Comment (cwjames1983) [inline papers/H0_I/Analysis/CRACO/Cloud/nautilus_craco_full_1.yaml:44]: If this is being merged back to H0_paper, better make sure you checkout H0_paper!

File: papers/H0_I/Analysis/CRACO/Cloud/nautilus_craco_full_10.yaml [hunk 2]
```diff
@@ -41,7 +41,7 @@ spec:
             python setup.py develop;
             cd ../zdm;
             git fetch;
-            git checkout H0_paper;
+            git checkout full_craco;
```
Comment (cwjames1983) [inline papers/H0_I/Analysis/CRACO/Cloud/nautilus_craco_full_10.yaml:44]: as per above

File: papers/H0_I/Analysis/CRACO/Cloud/nautilus_craco_full_2.yaml [hunk 3]
```diff
@@ -41,7 +41,7 @@ spec:
             python setup.py develop;
             cd ../zdm;
             git fetch;
-            git checkout H0_paper;
```
Comment (cwjames1983) [inline papers/H0_I/Analysis/CRACO/Cloud/nautilus_craco_full_2.yaml:44]: do you see a trend? :-)

File: papers/H0_I/Analysis/CRACO/Cloud/nautilus_craco_full_3.yaml [hunk 4]
```diff
@@ -41,7 +41,7 @@ spec:
             python setup.py develop;
             cd ../zdm;
             git fetch;
-            git checkout H0_paper;
```
Comment (cwjames1983) [inline papers/H0_I/Analysis/CRACO/Cloud/nautilus_craco_full_3.yaml:44]: I think even my 2nd year students could have guessed what I am going to say here! ;-)

File: papers/H0_I/Analysis/CRACO/Cloud/nautilus_craco_full_4.yaml [hunk 5]
```diff
@@ -41,7 +41,7 @@ spec:
             python setup.py develop;
             cd ../zdm;
             git fetch;
-            git checkout H0_paper;
```
Comment (cwjames1983) [inline papers/H0_I/Analysis/CRACO/Cloud/nautilus_craco_full_4.yaml:44]: ditto. Running out of witty remarks.

File: papers/H0_I/Analysis/CRACO/Cloud/nautilus_craco_full_5.yaml [hunk 6]
```diff
@@ -41,7 +41,7 @@ spec:
             python setup.py develop;
             cd ../zdm;
             git fetch;
-            git checkout H0_paper;
```
Comment (cwjames1983) [inline papers/H0_I/Analysis/CRACO/Cloud/nautilus_craco_full_5.yaml:44]: ditto

File: papers/H0_I/Analysis/CRACO/Cloud/nautilus_craco_full_6.yaml [hunk 7]
```diff
@@ -41,7 +41,7 @@ spec:
             python setup.py develop;
             cd ../zdm;
             git fetch;
-            git checkout H0_paper;
```
Comment (cwjames1983) [inline papers/H0_I/Analysis/CRACO/Cloud/nautilus_craco_full_6.yaml:44]: ditto

File: papers/H0_I/Analysis/CRACO/Cloud/nautilus_craco_full_7.yaml [hunk 8]
```diff
@@ -42,10 +41,9 @@ spec:
             python setup.py develop;
             cd ../zdm;
             git fetch;
-            git checkout H0_paper;
```
Comment (cwjames1983) [inline papers/H0_I/Analysis/CRACO/Cloud/nautilus_craco_full_7.yaml:45]: ditto

File: papers/H0_I/Analysis/CRACO/Cloud/nautilus_craco_full_8.yaml [hunk 9]
```diff
@@ -41,7 +41,7 @@ spec:
             python setup.py develop;
             cd ../zdm;
             git fetch;
-            git checkout H0_paper;
```
Comment (cwjames1983) [inline papers/H0_I/Analysis/CRACO/Cloud/nautilus_craco_full_8.yaml:44]: ditto

File: papers/H0_I/Analysis/CRACO/Cloud/nautilus_craco_full_9.yaml [hunk 10]
```diff
@@ -41,7 +41,7 @@ spec:
             python setup.py develop;
             cd ../zdm;
             git fetch;
-            git checkout H0_paper;
```
Comment (cwjames1983) [inline papers/H0_I/Analysis/CRACO/Cloud/nautilus_craco_full_9.yaml:44]: ditto

File: papers/H0_I/Analysis/CRACO/Explore_full.ipynb [hunk 11]
```diff
@@ -69,13 +69,22 @@
...
+       " 'pDMz',\n",
+       " 'pz',\n",
        " 'lEmax',\n",
        " 'H0',\n",
        " 'alpha',\n",
        " 'gamma',\n",
        " 'sfr_n',\n",
        " 'lmean',\n",
-       " 'lsigma']"
+       " 'lsigma',\n",
```
Comment (cwjames1983) [inline papers/H0_I/Analysis/CRACO/Explore_full.ipynb:82]: so if this is for the full cube, you'll want to add lls0...N0 for surveys 1,2,3,4 to the below

File: papers/H0_I/Analysis/CRACO/Explore_full.ipynb [hunk 12]
```diff
@@ -137,7 +146,7 @@
     {
      "data": {
       "text/plain": [
-       "-571.65"
```
Comment (cwjames1983) [inline papers/H0_I/Analysis/CRACO/Explore_full.ipynb:140]: Quick Q - what is this number?

**Review by cwjames1983 — CHANGES_REQUESTED** [review]
Just two changes. #1: you had better check out h0_paper rather than craco_full if we're going to pull this in (see all the comments on the nautilus files);
#2: for the real cube, you'll need to account for the five different surveys in Explore_full
#3: Git doesn't let me comment on unchanged parts of files, hence I can't make suggestions on the .json cube file, even though I would like to! Will do this is slack.

**Review by esanmouli — COMMENTED** [review]
Looks good!

**Pull request merged**